Sources/SentryCrash/Recording/Tools/SentryCrashSignalInfo.c

```diff
@@ -115,7 +121,10 @@ static const SentryCrashSignalInfo g_fatalSignalData[] = {
     SIGNAL_INFO_NOCODES(SIGPIPE),
     SIGNAL_INFO(SIGSEGV, g_sigSegVCodes),
     SIGNAL_INFO_NOCODES(SIGSYS),
-    SIGNAL_INFO(SIGTERM, g_sigTrapCodes),
+#if SIGTRAP_ENABLED
+    SIGNAL_INFO(SIGTRAP, g_sigTrapCodes),
+#endif
+    SIGNAL_INFO_NOCODES(SIGTERM),
```

Comment:
I think we can actually leave out the SIGTRAP_ENABLED macro and just allow SIGTRAP monitoring as well. This was fixed upstream in https://github.com/kstenerud/KSCrash/pull/370. Also, a few lines down, a comment explains that:
> Dereferencing a NULL pointer ... causes SIGTRAP, 0 on arm
So we probably want to catch those.

Reply:
You're right. I pulled that whole part out. So now we have the small bug fix and addition of SIGTERM.